Kontent.Ai.Management/Models/PreviewConfiguration/PreviewUrlPatternRequestModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Kontent.Ai.Management.Models.Shared;
+using Newtonsoft.Json;
+
+namespace Kontent.Ai.Management.Models.PreviewConfiguration;
+
+/// <summary>
+/// Represents preview URL pattern response model.
+/// </summary>
+public class PreviewUrlPatternRequestModel
```

Comment:
SDK does not use Request suffix. Also this model is identical to PreviewUrlPatternResponseModel. Can we unify it?

Reply:
removed and unified